client/state/memberships/coupon-list/test/schema.js

```diff
@@ -0,0 +1,65 @@
+import couponListSchema, { metadataSchema, metaKeytoSchemaKeyMap } from '../schema';
+
+describe( 'schema', () => {
```

Comment:
Schemas don't usually need a test, since what it does it just copies the original schema. We should be able to remove it unless you have a really good reason to keep it.